src/main/java/ladder/view/ResultView.java

```diff
@@ -0,0 +1,43 @@
+package ladder.view;
+
+import ladder.domain.*;
+import ladder.domain.enums.Direction;
+
+import java.util.Arrays;
+
+public class ResultView {
+
+    public void viewResult(Persons persons, Radder radder) {
+        System.out.println("실행결과\n");
+
+        viewPersons(persons);
+        viewRadder(radder);
+    }
+
+
+    private void viewPersons(Persons persons) {
+        Arrays.stream(persons.getNames())
+                .forEach(s -> System.out.print(String.format("%-" + (Person.MAX_NAME_LENGTH + 1) + "s", s)));
+        System.out.println("");
+    }
+
+    private void viewRadder(Radder radder) {
+        radder.getLines().stream().forEach(this::drawLine);
+    }
+
+    private void drawLine(Line line) {
+        line.getPoints().stream()
+                .forEach(x -> System.out.print("|" + drawHorizontal(x.equals(Direction.RIGHT))));
+        System.out.println("");
+    }
```

Comment:
```suggestion
        line.getPoints()
                .forEach(x -> System.out.print("|" + drawHorizontal(x.isNext())));
        System.out.println();
    }
```

- `.stream()` 없어도 됩니다 ^^
- x.isNext() 객체가 직접 비교하도록 (응집성이 좋아진다고 생각합니다.)

Reply:
사실 isNext 같은 역할을 하는 매소드 (`hasRIghtConnection`) 를 `VerticalLine` 일때 사용하다가, `Direction` 으로 통합하면서 굳이 필요할까? 라는 생각에 코드로 푼 부분이긴 합니다.
리뷰어님 의견에 동의하고, 의도를 명확히 하는 측면에서도 더 좋을 것 같네요